app/js/displayPageButtons.ts

```diff
@@ -60,11 +59,30 @@ function pageButtonCheck(page: number, pages: number) {
         } else {
             document.querySelector('.backBtn').classList.remove('greyedOutBtn');
         }
-
         if (page === pages) {
             document.querySelector('.nextBtn').classList.add('greyedOutBtn');
         } else {
             document.querySelector('.nextBtn').classList.remove('greyedOutBtn');
         }
     }
 }
+
+/**
+ * Shows or Hides the Pagination Buttons based on the boolean value passed in
+ * @param setToHidden boolean that determines if the Page Buttons are to be hidden or not
+ */
+function showPaginationButtons(setToHidden: boolean) {
+    let viewPageSelectors = document.querySelector('.pageSelectors');
+    if (setToHidden) {
+        viewPageSelectors.classList.add('hidden');
+    }
+    else {
+        // when there are over 20 users per page, then 4 Page Buttons are displayed (Previous, First Page, Second Page, Last) and in this case we show the whole pageSelectors class which contains those 4 buttons - without this the buttons are still displayed when there are less than 20 users - cannot use logic of number of user rows in table as that will  never be over 20
```

Comment:
can you shorten this comment to a single sentence thats less than the maximum line length